app/controllers/conversations_controller.rb

```diff
@@ -0,0 +1,29 @@
+class ConversationsController < ApplicationController
+
+  def show
+    @message = Message.new
+    @conversation = Conversation.find(params[:id])
+    @messages = @conversation.messages.order(:created_at)
+  end
+
+  def recent
+    @recent_conversations = Conversation.recently_active(25)
```

Comment:
Nice custom action.

I think I'd probably just call this the index but still only get the top 25 active conversations.